core/src/main/java/com/klaytn/caver/tx/kct/KIP7.java

```diff
@@ -0,0 +1,787 @@
+     * Instance method for calling method "burn" in KIP-7 Standard Token
+     *
+     * @param amount The token amount to be burned
+     * @return RemoteCall
+     */
+    public RemoteCall<KlayTransactionReceipt.TransactionReceipt> burn(BigInteger amount) {
+        final Function function = new Function(
+                FUNC_BURN,
+                Arrays.<Type>asList(new org.web3j.abi.datatypes.generated.Uint256(amount)),
+                Collections.<TypeReference<?>>emptyList());
+        return executeRemoteCallTransaction(function);
+    }
+
+    /**
+     * Instance method for calling method "burnFrom" in KIP-7 Standard Token
+     *
+     * @param account The account will be deducted is the The token amount to be burned
+     * @param amount The token amount to be burned
+     * @return RemoteCall
+     */
+    public RemoteCall<KlayTransactionReceipt.TransactionReceipt> burnFrom(String account, BigInteger amount) {
+        final Function function = new Function(
+                FUNC_BURNFROM,
+                Arrays.<Type>asList(new org.web3j.abi.datatypes.Address(account),
+                        new org.web3j.abi.datatypes.generated.Uint256(amount)),
+                Collections.<TypeReference<?>>emptyList());
+        return executeRemoteCallTransaction(function);
+    }
+
+    /**
+     * Instance method for calling method "decimals" in KIP-7 Standard Token
+     *
+     * @return RemoteCall
+     */
+    public RemoteCall<BigInteger> decimals() {
+        final Function function = new Function(FUNC_DECIMALS,
+                Arrays.<Type>asList(),
+                Arrays.<TypeReference<?>>asList(new TypeReference<Uint8>() {}));
+        return executeRemoteCallSingleValueReturn(function, BigInteger.class);
+    }
+
+    /**
+     * Instance method for calling method "isMinter" in KIP-7 Standard Token
+     *
+     * @param account The account to check the minting permission
+     * @return RemoteCall
+     */
+    public RemoteCall<Boolean> isMinter(String account) {
+        final Function function = new Function(FUNC_ISMINTER,
+                Arrays.<Type>asList(new org.web3j.abi.datatypes.Address(account)),
+                Arrays.<TypeReference<?>>asList(new TypeReference<Bool>() {}));
+        return executeRemoteCallSingleValueReturn(function, Boolean.class);
+    }
+
+    /**
+     * Instance method for calling method "isPauser" in KIP-7 Standard Token
+     *
+     * @param account The account to check the pausing permission
+     * @return RemoteCall
+     */
+    public RemoteCall<Boolean> isPauser(String account) {
+        final Function function = new Function(FUNC_ISPAUSER,
+                Arrays.<Type>asList(new org.web3j.abi.datatypes.Address(account)),
+                Arrays.<TypeReference<?>>asList(new TypeReference<Bool>() {}));
+        return executeRemoteCallSingleValueReturn(function, Boolean.class);
+    }
+
+
+    /**
+     * Instance method for calling method "mint" in KIP-7 Standard Token
```

Comment:
How about adding explaination about permission? mint and pause need specific permission. (include addMinter, addPauser)

Reply:
Okay, I'll add comment.